usr/share/rear/layout/prepare/Linux-s390/370_confirm_dasd_format_code.sh

```diff
@@ -0,0 +1,69 @@
+# adapted from 100_confirm_layout_code.sh
+#
+# Let the user confirm the
+# DASD format code (dasdformat.sh) script.
+#
+
+is_false "$FORMAT_DASDS" && return 0
+
+# Show the user confirmation dialog in any case but when not in migration mode
+# automatically proceed with less timeout USER_INPUT_INTERRUPT_TIMEOUT (by default 10 seconds)
+# to avoid longer delays (USER_INPUT_TIMEOUT is by default 300 seconds) in case of unattended recovery:
+# (taken from 120_confirm_wipedisk_disks.sh)
+local timeout="$USER_INPUT_TIMEOUT"
+is_true "$MIGRATION_MODE" || timeout="$USER_INPUT_INTERRUPT_TIMEOUT"
+
+rear_workflow="rear $WORKFLOW"
+original_disk_space_usage_file="$VAR_DIR/layout/config/df.txt"
+rear_shell_history="$( echo -e "cd $VAR_DIR/layout/\nvi $DASD_FORMAT_CODE\nless $DASD_FORMAT_CODE" )"
+unset choices
```

Comment:
better define like this
```
local choices=( 
"line 1"
"line 2"
...)
```

or, once we decide to do #2941 `mapfile -t choices <<EOF` makes this much simpler

Reply:
That is a copy of my code
that I used at several places,
cf. what
```
# find usr/sbin/rear usr/share/rear -type f | xargs grep -A2 'unset choices'
```
shows like
```
unset choices
choices[0]="fist choice"
choices[1]="second choice"
...
if CONDITION ; then
   choices[1]="another second choice" 
...
        case "$choice" in
            (${choices[0]})
               ...
            (${choices[1]})
...
```
e.g. see
layout/recreate/default/200_run_layout_code.sh
where choices are rather complicated.

I have no idea why I did not use a local array.

I think I used the explicit choices[0], choices[1]
in the initial assingments to make it more clear
what each choice in the subsequent code is.